extensions/amp-story/1.0/amp-story.css

```diff
@@ -279,6 +291,13 @@ amp-story-grid-layer {
   pointer-events: none !important;
 }
 
+:root:not(.i-amphtml-story-vertical) amp-story, amp-story:not(.i-amphtml-story-loaded) amp-story-page:first-of-type,
```

Comment:
Plz add a less short comment :))

Reply:
I was able to simplify the CSS rules here so the comment doesn't have to be that long anymore 😁